packages/shared/src/hooks/auth/useOnboarding.tsx

```diff
@@ -9,9 +6,7 @@ interface UseOnboarding {
 
 export const useOnboarding = (): UseOnboarding => {
   const { isAuthReady, user } = useAuthContext();
-  const postPageOnboarding = useFeature(feature.postPageOnboarding);
-  const shouldShowAuthBanner =
-    postPageOnboarding === PostPageOnboarding.V1 && isAuthReady && !user;
+  const shouldShowAuthBanner = isAuthReady && !user;
```

Comment:
So it should always be shown I guess. We can remove the `PostFeedFiltersOnboarding` component then?

Reply:
Yes the banner will always be shown on post page when user is not authenticated. But hmm not sure, which component is that and what does it do?

Reply:
This is what I was referring to that we need to clean up. The banner on top of the post page/modal that we matched this experiment against.